readthedocs/templates/builds/build_detail.html

```diff
@@ -174,6 +174,19 @@
             {% endblocktrans %}
           </p>
         </div>
+      {% else %}
+        <div class="build-ideas">
+          <p>
+            {% blocktrans trimmed %}
+              <strong>Your build is properly using a configuration file ✅</strong><br/>
+              {% if build.readthedocs_yaml_path %}}
+                The configuration file used was: <code>{{ build.readthedocs_yaml_path }}</code>.
+              {% else %}
+                The configuration file used was: <code>.readthedocs.yaml</code>.
```

Comment:
Be careful with this 😄 . Note that it could have been `readthedocs.yaml` or `readthedocs.yml` or `.readthedocs.yml`. We don't know at this point 🙃 

Maybe we should not mention this here and just output the `cat <path-to-yaml>` as you mentioned in Slack for the case where the config file was found.

Reply:
Yeah, I'd opt for putting this in the build output as well. I've been trying to get us away from making one-off UI blocks like this as they tend to overload the page design. I think putting it in the build output as `cat .readthedocs.yaml` makes sense and gives us a command to point the user to.

It also serves as a way to show the user what configuration was detected. The beta dashboard shows the rendered version in addition, but I commonly want to see both.

Reply:
Also, I'll echo feedback I had on this in another issue: I'd like to user fewer patterns to give the user feedback and ideally stick to a single pattern. For now, this pattern has been integrating the debug output in the command listing.

We did talk about how to redo our notification system, which is work I keep raising. I'll continue this discussion in a new issue and hopefully we can put some priority on it.